pkgs/tools/games/pyroveil/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  stdenv,
+  cmake,
+  lib,
+  ninja,
+  gcc,
+  git,
+  python311,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  pname = "Pyroveil";
+  # I don't see a version specification anywhere, only in json files.
+  version = "2.0.0";
+  src =
+    (fetchFromGitHub {
+      owner = "HansKristian-Work";
+      repo = "pyroveil";
+      rev = "7fde4d6d0ebc664f33ebc6a42d8c2975bf353bfe";
+      sha256 = "sha256-sKvqcmQsbK8YuCdN2+5NKmyE0Sc8iA45IpYnFRItqiU=";
+      fetchSubmodules = true;
+    }).overrideAttrs
+      (oldAttrs: {
+        env = oldAttrs.env or { } // {
+          GIT_CONFIG_COUNT = 1;
+          GIT_CONFIG_KEY_0 = "url.https://github.com/.insteadOf";
+          GIT_CONFIG_VALUE_0 = "git@github.com:";
+        };
+      });
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+    gcc
+    git
+    python311
```

Comment:
Does it only build with Python 3.11? Usually we would use the default `python3` package